maintainers/maintainer-list.nix

```diff
@@ -9617,6 +9617,12 @@
     githubId = 144952;
     name = "Suvash Thapaliya";
   };
+  svavs = {
+    name = "Silvano Sallese";
+    email = "silvano.sallese+nixpkgs@gmail.com";
+    github = "svavs";
+    githubId = 1369198;
+  };
```

Comment:
Separate commit: `maintainers: add svavs`

Reply:
Done. Thank you.